File: Python/python day 1/exercise3.py
import math
a = int(input("Add the value for side one:"))
b = int(input("Add the value for side two:"))
c = int(input("Add the value for side one:"))
r = 0
r = (a + b + c)*(b + c - a)*(a - b + c)*(a + b - c)
x = 1/4 * math.sqrt(r)
print(x)

File: Java/Python1ToJava/Exercise3.java
import java.util.Scanner;
public class Exercise3{
  public static void main(String args[]){
    int [] array = new int[3];
      for(int i=1; i<4; i++){
        System.out.print("Add the value for side " +i+" :");
        Scanner number = new Scanner(System.in);
        array[i-1] = number.nextInt();
      }
    double r = 0;
    r = (array[0]+array[1]+array[2])*(array[1]+array[2]-array[0])*(array[0]-array[1]+array[2])*(array[0]+array[1]-array[2]);
    double a = (0.25) * Math.sqrt(r);
    System.out.println(a);
  }
}
